src/borg/archiver.py

```diff
@@ -2643,6 +2653,10 @@ def define_borg_mount(parser):
                                 help='stay in foreground, do not daemonize')
             parser.add_argument('-o', dest='options', type=str,
                                 help='Extra mount options')
+            parser.add_argument('--bypass-lock', dest='lock',
+                                action='store_false',
+                                default=argparse.SUPPRESS,
```

Comment:
is this the only place where we use SUPPRESS? had to look it up in argparse docs.

maybe add a comment like "only create args.lock attribute if this option is specified".

Reply:
Indeed, it is. Didn't expect that, I'm using `SUPPRESS` quite often. Will add your suggested comment, can't hurt. :+1: